chapter_introduction/index_vn.md

```diff
@@ -2549,9 +2549,9 @@ Những công cụ này cho phép các trạng thái nội tại của mạng n
 
 * Một bước phát triển quan trọng khác là sự ra đời của GAN :cite:`Goodfellow.Pouget-Abadie.Mirza.ea.2014`.
 Trong quá khứ, các phương pháp thống kê để đánh giá hàm mật độ xác suất và các mô hình sinh (_generative models_) tập trung vào việc tìm các phân phối xác suất hợp lý và các thuật toán (thường là xấp xỉ) để lấy mẫu từ các phân phối đó.
-Kết quả là, những thuật toán này cũng bị hạn chế bởi sự thiếu linh động do kế thừa từ chính các mô hình thống kê đó.
-Phát kiến quan trọng của GAN là thay thế thuật toán lấy mẫu bất kỳ bởi các tham số khả vi [có thể tối ưu được dựa vào các phương pháp tối ưu dựa trên đạo hàm].
-Các phương pháp này sau đó được điều chỉnh sao cho bộ phân loại (có hiệu quả giống như bài kiểm tra hai mẫu trong xác suất) không thể phân biệt giữa dữ liệu thật và giả.
+Vì vậy, những thuật toán này có rất nhiều hạn chế bởi sự thiếu linh động của việc kế thừa từ chính các mô hình thống kê đó.
+Phát kiến quan trọng của GANs là thay thế thuật toán lấy mẫu bất kỳ bởi các tham số khả vi [có thể tối ưu được dựa vào các phương pháp tối ưu dựa trên đạo hàm].
```

Comment:
```suggestion
Phát kiến quan trọng của GANs là thay thế thuật toán lấy mẫu bất kỳ bởi các tham số khả vi (có thể tối ưu được dựa vào các phương pháp tối ưu dựa trên đạo hàm).
```

Reply:
Phần trong ngoặc này mình được anh Tiệp giải thích là nội dung trong đó được hiểu là góc nhìn của người dịch. Tuy nhiên thì bản thân văn bản trong ngoặc vuông này được quy định sử dụng khi muốn chỉ về một nội dung có dẫn nguồn, tức là các nội dung có đường dẫn references.

Bản thân các nội dung khi mình dịch cũng đã đoán ý tác giả khá nhiều, nên cái này mình thống nhất là ý của người dịch thì cứ mở ngoặc tròn ra để hiểu đây là phần giải thích từ người dịch hoặc là một phần thông tin thêm nhằm làm rõ ý của toàn đoạn văn.